docs/General/Meetings/Radiomics_meeting.md

```diff
@@ -0,0 +1,16 @@
+# Radiomics Meeting
+## Time and Room
+Radiomics meeting happens every 2 weeks on Tuesday, 2:00 - 3:00pm at PMCRT 11-710
```

Comment:
For general formatting, it would be a good idea to add periods after each sentence.

```suggestion
Radiomics meeting happens every 2 weeks on Tuesday, 2:00 - 3:00pm at PMCRT 11-710.
```